axoSlackBot.js

```diff
@@ -46,7 +46,7 @@ controller.setupWebserver(config.port,function(err,webserver) {
         var params = {
           grant_type: "authorization_code",
           code: code,
-          redirect_uri: config.redirectUri+ "authorizationCode",
+          redirect_uri: config.redirectUri.substring(0, config.redirectUri.indexOf("oauth")) + "authorizationCode",
```

Comment:
@BrettGAxosoft because when I just add / before  authorizationCode, we will get authorization failed from axosoft. that's why I used substring